.github/workflows/e2e-build-images.yaml

```diff
@@ -30,12 +31,20 @@ jobs:
         run: |
           set -eu
 
-          all="$(distro-info --supported-esm) $(distro-info --supported)"
-          all="$(echo $all | tr ' ' '\n' | sort -u)"
+          all="$(distro-info --supported-esm --fullname)\n$(distro-info --supported --fullname)\n"
+          all="$(printf "$all" | sort -u)"
 
           releases=""
+          versions=""
+
+          while IFS= read -r release; do
+              # Extract the version and codename
+              version=$(echo "$release" | awk '{print $2}')
+              r=$(echo "$release" | sed -E 's/.*"([^"]+)".*/\1/' | awk '{print tolower($1)}')
```

Comment:
couldn’t we use distroinfo with codename to feed that argument? I know that will force it to run twice to extract version and codename (because it seems options don’t combine :/), but we can assume they are in the same order?